### PR TITLE
Fix selfLinks in test examples to v1beta3

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

**Comment by wojtek-t** [comment]
cc @nikhiljindal

File: pkg/api/replication_controller_example.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "metadata": {
         "name": "elasticsearch-logging-controller",
         "namespace": "default",
-        "selfLink": "/api/v1beta1/replicationControllers/elasticsearch-logging-controller?namespace=default",
+        "selfLink": "/api/v1beta3/namespaces/default/replicationControllers/elasticsearch-logging-controller",
```
Comment (bgrant0607) [inline pkg/api/replication_controller_example.json]: replicationcontrollers should be all lowercase

Comment (wojtek-t) [inline pkg/api/replication_controller_example.json]: Done.

**Comment by bgrant0607** [comment]
LGTM, thanks

**Pull request merged**